mineral-ui/mineral-ui
# [utils] createThemedComponent only uses ThemeProvider to pass custom theme

### Description

In `createThemedComponent`, we’re passing along the custom theme to be applied to both the `ThemeProvider` used by the helper _and_ the wrapped component. Per [glamorous docs](https://glamorous.rocks/advanced#theming) (the end of the code block), the surrounding theme is ignored when you pass a theme directly to a component via props. So, this change is to _not_ pass the custom theme directly to the wrapped component.

This PR also:
- Updates the Sample component to use proper theme variables
- Updates the `utils` examples to be more clear and helpful
- Updates snapshots

### Motivation and context

`createThemedComponent` currently does not work on any Mineral UI components. This fixes that.

### How to test


1. `cd mineral-ui && npm start`
1. Navigate to utils
1. Confirm that the `createThemedComponent` example is working correctly
1. Open that example in your editor
    1. Change Sample to Button (remember to import and add to `scope`)
    1. Confirm that the example still works as expected
    1. (Note: all of the above is tested via snapshots)

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist




### How does this PR make you feel?

![Repair man man man](https://media.giphy.com/media/5OJjiJR1ujaAU/giphy.gif)
